modules/core/arrow-core-data/src/test/kotlin/arrow/core/SortedMapKTest.kt

```diff
@@ -24,9 +27,9 @@ class SortedMapKTest : UnitSpec() {
   }
 
   init {
-
     testLaws(
       EqLaws.laws(SortedMapK.eq(String.eq(), Int.eq())) { sortedMapOf("$it" to it).k() },
+      HashLaws.laws(SortedMapK.hash(String.hash(), Int.hash()), EQ) { sortedMapOf("key" to it).k() },
```

Comment:
HashLaws already include EqLaws, so you can skip the line above :D https://github.com/arrow-kt/arrow/blob/master/modules/core/arrow-test/src/main/kotlin/arrow/test/laws/HashLaws.kt#L11